examples/plot_9_multi_image_separate.py

```diff
@@ -0,0 +1,52 @@
+"""
+Force plots to be displayed on separate lines
+=============================================
+This example demonstrates how the visualisation of multiple plots produced from a single
+code block can be controlled. The default behaviour is to stack plots side-by-side,
+however this can be overridden to display each plot created by the code block on a
+separate line, preserving their size.
```

Comment:
Added a new example demonstrating the `sphinx_gallery_multi_image` and `..._block` options.